templates/workspace_services/guacamole/user_resources/guacamole-azure-export-reviewvm/porter.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 schemaVersion: 1.0.0
 name: tre-service-guacamole-export-reviewvm
-version: 0.2.2
+version: 1.0.0
```

Comment:
> Bump the Porter version to indicate a breaking change https://github.com/microsoft/AzureTRE/issues/4421#issuecomment-2740633545

Don't you want to enable easier upgrade to this version?
This version includes an ignore change for custom data, so what "damage" can come from upgrading the bundle?

Reply:
This is in response to [this comment from Marcus]()https://github.com/microsoft/AzureTRE/issues/4421#issuecomment-2740633545

Reply:
Yes, that's true, its fine not to be a major upgrade.  Sorry.